roles/include_components/README.md

```diff
@@ -1,15 +1,46 @@
-# include_components role
+# redhat.ocp.include_components role - create and attach DCI components to the DCI jobs
 
-The `include_components` role adds components in the DCI jobs from software resources available in the ansible controller node, for example: rpm packages, and git repositories used during the job execution.
+- [Synopsis](https://galaxy.ansible.com/ui/repo/published/redhatci/ocp/content/role/include_components/README.md#synopsis)
+- [Parameters](https://galaxy.ansible.com/ui/repo/published/redhatci/ocp/content/role/include_components/README.md#parameters)
+- [Examples](https://galaxy.ansible.com/ui/repo/published/redhatci/ocp/content/role/include_components/README.md#examples)
```

Comment:
:disappointed:  yeah I inspected the doc generated for the roles in galaxy and I don't see any anchors in there, I don't think the links added in here will work either :( Take as an example: 

https://galaxy.ansible.com/ui/repo/published/redhatci/ocp/content/role/acm_hypershift/README.md#variables

I guess we will have to use GH links

Reply:
@tonyskapunk your link shows "404 - Page not found"
I checked the galaxy page for [acm_hypershift](https://galaxy.ansible.com/ui/repo/published/redhatci/ocp/content/role/acm_hypershift) role, there is [just one link in-there](https://galaxy.ansible.com/ui/repo/published/redhatci/ocp/content/acm_setup/README.md), and it's broken.

Reply:
Let's use full links for the moment, [as the others do](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/copy_module.html) for the ansible documentation.

Reply:
> @tonyskapunk your link shows "404 - Page not found" I checked the galaxy page for [acm_hypershift](https://galaxy.ansible.com/ui/repo/published/redhatci/ocp/content/role/acm_hypershift) role, there is [just one link in-there](https://galaxy.ansible.com/ui/repo/published/redhatci/ocp/content/acm_setup/README.md), and it's broken.

Correct, that's my point :smile: this is to mimic the links in the README added in this change, they will also show a 404 if you leave them in there.

Reply:
oh I see now, I'll think how to fix that

Reply:
@tonyskapunk Since it's a generic problem with galaxy documentation, not related to the scope of a current PR, I decided to remove this part. I cannot figure out right away how to fix that, and it's been one week since this merge has been pending because of CI and ansible-lint issues. 

We could address the documentation formatting later, in a separate change.